site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -177,14 +177,17 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `agones.allocator.replicas`                              | The number of replicas to run in the deployment                                                                                                                                                                         | `3`                                |
 | `agones.allocator.service.name`                          | Service name for the allocator                                                                                                                                                                                          | `agones-allocator`                 |
 | `agones.allocator.service.serviceType`                   | The [Service Type][service] of the HTTP Service                                                                                                                                                                         | `LoadBalancer`                     |
+| `agones.allocator.service.http.nodePort`, `agones.allocator.service.grpc.nodePort`                   | The [NodePort] serves as the external entry point for incoming requests for your app                                                                                                                                                                         | `30000-32767`                     |
```

Comment:
Let's not combine the two, but have separate rows for each please.

We should also make clear - This parameter is only used if the ServiceType is set to "NodePort".

Thanks!

Reply:
These two parameters are added in separate rows. Thanks!